server/dfr3-service/src/main/java/edu/illinois/ncsa/incore/service/dfr3/models/DFR3Set.java

```diff
@@ -38,13 +40,8 @@
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
     private List<String> spaces;
 
-    // region Getters
     public String getId() {
-        if (id == null) {
-            return null;
-        } else {
-            return id.toHexString();
```

Comment:
Hmmm, was this a bug?

Reply:
I should rephrase, wonder why we did it since toString and toHexString return the same value, but curious why we had hex.

Reply:
I guess it's just different developers working on different services and using their own convention. There is lot of such inconsistencies between space and data services vs DFR3 and hazard services.